src/main/java/uk/gov/hmcts/darts/cache/token/service/value/impl/ServiceContextCacheValue.java

```diff
@@ -76,8 +76,8 @@ public ServiceContext getServiceContext() throws CacheException {
         return context;
```

Comment:
Think I've mentioned this before - `JAXBContext.newInstance` in the 2 places here, should probably be changed to be in one place only as it's an expensive operation. Can it be done in the default constructor like in `SoapHeaderConverter`?